### PR TITLE
Add stable version for Data Share and in place sharing support for Kusto

### PR DESCRIPTION
**Pull request by ghost**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by anthony-c-martin** [comment]
This PR mostly consists of changes which have already been signed off on in azure-rest-api-specs-pr. I'm signing off on the introduction of new stable version + additional "sourceShareLocation" field.

**Comment by sarajang** [comment]
@erich-wang  please merge this PR when you get a chance. thanks.

File: specification/datashare/resource-manager/readme.go.md [hunk 1]
```diff
@@ -23,4 +24,13 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
...
 output-folder: $(go-sdk-folder)/services/preview/datashare/mgmt/2018-11-01-preview/$(namespace)
+```
+
+### Tag: package-2019-11-01 and go
+
+These settings apply only when `--tag=package-2019-11-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2019-11-01' && $(go)
+output-folder: $(go-sdk-folder)/services/stable/datashare/mgmt/2019-11-01/$(namespace)
```
Comment (ArcturusZhang) [inline specification/datashare/resource-manager/readme.go.md]: Please change this to
`$(go-sdk-folder)/services/datashare/mgmt/2019-11-01/$(namespace)`

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]
Please change the `output-folder` in `readme.go.md`, which will cause a CI failure in go SDK. Thanks!

File: specification/datashare/resource-manager/readme.go.md (hunk 1 above)
Comment (sarajang) [reply specification/datashare/resource-manager/readme.go.md]: @ArcturusZhang  updated. thanks.

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**